### DISCUSSION TITLE
Wrong page display

### DISCUSSION THREAD

**Discussion by andy9a9** [community/community#30267] [Mobile] [answered]
Dear support!
I have found that https://github.com/Homebrew/homebrew-cask is wrongly displayed on mobile device. Looks like that image is not correctly scaled -> wrong page size. 
![Screenshot_2022-08-21-09-15-35-558_com hsv freeadblockerbrowser](https://user-images.githubusercontent.com/20794139/185780102-92fdc985-263a-4ffc-a3e8-8ac1ff3c8c14.jpg)

Hopefully this is the right place for posting a bug :).

**Comment by radar-blip** [comment, marked as answer]
This might be a better place: [Bug-Report](https://support.github.com/contact/bug-report)

**Reply by andy9a9** [reply]
Thanks, I will move it there.

**Reply by radar-blip** [reply]
No problem. Good Luck!
